### ISSUE TITLE
Squash "access to protected member of a class" warnings in cint API

### ISSUE THREAD

**Issue by Utumno** [wrye-bash/CBash#17]
Pycharm emits no less than 120 warnings _outside cint_ on _RecordID, _Type (Type ?), and _ModID attributes access. Although as we discussed PEP8'ing the API is not an immediate priority (although should be done in the final API), those in particular should be squashed in the next API iteration (and _Type renamed to something more descriptive).

In general I have painfully and meticulously squashed 1000s of warnings and those were of the most annoying kind, as I could not/would not mess with cint. I will be very glad to see those squashed.

**Comment by leandor** [comment]
Yeah, you mean these kind of warnings, right?

![image](https://cloud.githubusercontent.com/assets/1095373/19822801/b8fc332e-9d3b-11e6-98bd-ccb881186289.png)

I believe those are totally an artifact of the way cint wraps the C interface today, I'd hope all of that won't be needed at all when the actual API can be published in full.

Of course, it may mean having to do some changes on the code base, if/when there is code that depends on those actual wrappers existing... I'll try to mimic them the best I can, and if so, I'll totally provide a more clear interface w.r.t. names and style.

**Comment by Utumno** [comment]
Yep - underscore prepended names in python are considered private - a convention mainly which I have been putting into good use since I started refactoring. So all those slots should not start with an underscore - and then be lowercase (cause CamelCase is for types in python).

Just a head's up, probably will just disappear with cint :)
